app/main/routes.py

```diff
@@ -19,6 +20,7 @@ def before_request():
         db.session.commit()
         g.search_form = SearchForm()
     g.locale = str(get_locale())
+    db.create_all()
```

Comment:
Without this Archive was not being found, documentation showed to put this in this function which was already in the base repository.

Reply:
what do you mean `archive was not being found` do you mean the archive table in the database?

Reply:
Yes, when compiling an error was being thrown that the Archive was not a table even though it very much was and was even imported above.

Reply:
ORM != database table. creating tables like that requires migrations and so should be a database migration. If i understand it correctly, every time a request comes in, it will try to recreate the entire database

Reply:
![Screenshot 2023-03-15 223915](https://user-images.githubusercontent.com/35317693/225496085-e148f7ea-a4eb-4063-9af5-e3f87e764512.png)
There error I get while running without this line.

Reply:
Should this maybe be somewhere else? The db is first imported in this file though.

Reply:
yes that is because that database table does not exist. an ORM class is not a database table and does not guarantee making it. That is what database migrations are for. Please read on how to do it here https://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-iv-database

If i understand it correctly, every time a request comes in, it will try to recreate the entire database. trying to recreate the entire database every time a request comes in will be super slow compared to without that.